apps/src/templates/instructions/CommentArea.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  textInput: {
+    marginTop: 0,
+    marginBottom: 16,
+    display: 'block',
+    width: '90%'
+  },
+  textInputStudent: {
+    margin: 10,
+    display: 'block',
+    width: '90%',
+    backgroundColor: color.lightest_cyan
+  },
+  h1: {
+    color: color.charcoal,
+    marginTop: 8,
+    marginBottom: 12,
+    fontSize: 24,
+    fontFamily: '"Gotham 5r", sans-serif',
+    fontWeight: 'normal'
+  }
+};
```

Comment:
> I am feeling some guilt about the guidance I've given you on extracting the `CommentArea`, because what you've ended up extracting there does not greatly reduce the amount of code needed in this file to define the comment area.

you are probably tired of hearing me discuss this... but when you factor in all the `styles` specific to `CommentArea` that come with it, extracting `CommentArea` does seem to help clean up the file it came from.

Reply:
I agree. The list of styles was getting crazy so breaking it up felt good.